frame/instrumental-strategy-pablo/src/mock/helpers.rs

```diff
@@ -0,0 +1,47 @@
+use composable_traits::{defi::CurrencyPair, dex::Amm};
+use frame_support::{assert_ok, traits::fungibles::Mutate};
+use pallet_pablo::PoolInitConfiguration;
+use primitives::currency::CurrencyId;
+use sp_runtime::Permill;
+
+use crate::mock::{
+	account_id::{AccountId, ALICE, BOB},
+	runtime::{Balance, BlockNumber, Pablo, PoolId, Tokens},
+};
+
+pub fn create_usdt_usdc_pool() -> PoolId {
```

Comment:
`CurrencyId` from `primitives` crate sadly doesn't have `ETH`, `BTC` or something like native currency from another blockchains. Sure we can create mock assets for tests, but it will be integers (1, 2, 3) too, so there will be no difference for the tests.

Reply:
That's great

Reply:
Maybe, USDT/USDC ok too, but I just don't sure

Reply:
Temporary fixed by 2609356.